doc/whats_new/v1.0.rst

```diff
@@ -263,6 +263,12 @@ Changelog
   :pr:`18649` by `Leandro Hermida <hermidalc>` and
   `Rodion Martynov <marrodion>`.
 
+- |Fix| The `fit` method of the successive halving parameter search 
+  (:class:`HalvingGridSearchCV`, and :class:`HalvingRandomSearchCV`) now 
+  correctly handles `groups` parameter that supports the grouped labels for 
+  the samples used while splitting the dataset into train/test set. 
```

Comment:
Small suggestion to add a link, and to shorten it a bit:

```suggestion
  (:class:`model_selecion.HalvingGridSearchCV`, and :class:`model_selection.HalvingRandomSearchCV`) now 
  correctly handles thd `groups` parameter. 
```

Reply:
Yep. Changed in the latest commit.